### ISSUE TITLE
RLP CLI not working on full release

### ISSUE THREAD

**Issue by chrissyhroberts** [opendatakit/briefcase#719]
Using the full release of v1.14.0
CLI version of the resume last pull function does not work correctly. 

```
java -jar ODK-Briefcase-v1.14.0.jar -plla -rlp -pp -e -id form -f form.csv -sd xmls -url https://foo.bar/foo -u admin -p password -ed output -pf key.pem -oc  

Error: Unrecognized option: -rlp
```

Going back to beta release fixes the problem

```
java -jar ODK-Briefcase-v1.14.0-Beta.0.jar -plla -rlp -pp -e -id form -f form.csv -sd xmls -url  https://foo.bar/foo -u admin -p password -ed output -pf key.pem -oc

Feb 22, 2019 9:37:45 AM org.hsqldb.persist.Logger logInfoEvent
INFO: checkpointClose start
...

...
Exporting form form (form) to: output
```

**Comment by yanokwa** [comment]
We changed the flag to `-sfl` or `--start_from_last` because that felt clearer. We put the change in the release notes and the docs. If that flag works, please close the issue.

**Comment by chrissyhroberts** [comment]
facepalm! 
Thanks Yaw.

**Comment by ggalmazor** [comment]
Thanks for jumping in with the solution, @yanokwa!

**Comment by ggalmazor** [comment]
Hi, @chrissyhroberts!

It's been some days since v1.14.0 and I wanted to ask you for any feedback you can give us in case you've been using it for the last couple of weeks :)

**Comment by chrissyhroberts** [comment]
Looks good to me so far. Using it for all my big projects and it seems to work perfectly. SFL function is brilliant and seems robust. Running automated daily pulls and my progress charts haven't highlighted anything like whole days missing and so on. 

I did wonder where the SFL info is stored. Presumably this is within the data folder and not in the app itself. Also might be interesting to think about some kind of user accessible report on this. i.e. a file that says 'form name = xxxxx', total number of records = n, chunks = n, last chunk pulled = n', 'last pull date = yyyy-mm-dd'
Perhaps it already exists, but something like this would make quick checks on progress much easier for people using CLI in particular.

**Comment by ggalmazor** [comment]
Thanks for the feedback, @chrissyhroberts!

The SFL cursor is stored using the Java Preferences system. Java uses some XML documents to store information somewhere in the user's computer. The location varies depending on the operating system.

User feedback (on the CLI and the UI) is a longstanding issue we need to address, but odds are that a full review won't come before v2.0, although maybe we could add some simple feedback for SFL on the CLI.

I think the next useful thing could be to add a new CLI arg that lets you define the date from which you want to start pulling subs.

**Comment by chrissyhroberts** [comment]
I think that a CLI arg to define date would be most useful. 
Use cases

* I need to copy an ODK storage folder to another machine (breakdown, task delegation, restore from backup, balancing CPU loads) and would have to start from zero each time because RLP is in java folder on old machine
* I go away and the internet is really slow, so I ask my colleagues to pull specific date range on their machine, then to send me exported CSV which I can stitch to my current one.

**Comment by yanokwa** [comment]
@ggalmazor I think the v2 database will solve the issue of moving an install between machine. And yeah, adding a date range would be a good feature!

**Comment by ggalmazor** [comment]
Hi, @chrissyhroberts! You might want to try out the new v1.15.0 release which adds a new CLI arg to "start from date". The date would match the internal Aggregate update date of a submission, which corresponds to the date when it was last inserted or updated in its database.

**Comment by chrissyhroberts** [comment]
Thanks @ggalmazor. I'll have a look at it as soon as I get a chance. Thinking about this, now that we have three great pull/export features ```start-from-last``` and ```start-from-date``` and ```export-from-date``` what's now logically missing is ```export from last``` where briefcase looks at the target csv file, checks the last exported record and adds just the new ones.  

SFL has saved us about 15-20 hours a day pulling, but export time is now becoming an issue (i.e. deleting old csv and exporting all data to a new one). At present we are exporting ~750,000 forms to csvs and this is taking about 5-7 hours.  

This behaviour can of course already be handled using ```export from date```, except that if we hadn't pulled for a few days, we'd have to somehow store/read the last export date and if I exported up to say midnight this morning, then I'd leave off newest data until tomorrow's reports or risk duplication at the next pull.  Also only works at granular level of the day, so if we pulled twice a day we would potentially miss or duplicate some records. An ```export from last``` would have a fairly good use case for long lived forms, especially when managed on CLI
